<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadWaitForm</class>
 <widget class="QWidget" name="UploadWaitForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(179, 225, 255, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget
{
	border-image: url(:/resource/images/common/Transparent_Black_80.png);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>260</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1025</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QWidget" name="widgetWaitBg" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>1025</width>
             <height>150</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>150</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QWidget#widgetWaitBg
{
	background-color: rgba(255, 255, 255, 0);
}</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QWidget" name="widgetWait" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>1025</width>
                <height>141</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>141</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QWidget#widgetWait
{
	background-color: rgb(255, 255, 255);
}</string>
              </property>
              <widget class="QWidget" name="widget_4" native="true">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>20</y>
                 <width>701</width>
                 <height>91</height>
                </rect>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>541</width>
                 <height>91</height>
                </size>
               </property>
               <widget class="Label" name="labelString">
                <property name="geometry">
                 <rect>
                  <x>190</x>
                  <y>0</y>
                  <width>161</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Nexa Light</family>
                  <pointsize>17</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                  <stylestrategy>PreferDefault</stylestrategy>
                  <kerning>true</kerning>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(82, 164, 225);</string>
                </property>
                <property name="text">
                 <string>Please Wait...</string>
                </property>
               </widget>
               <widget class="Label" name="labelState">
                <property name="geometry">
                 <rect>
                  <x>190</x>
                  <y>30</y>
                  <width>501</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>微软雅黑</family>
                  <pointsize>11</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Compiling...</string>
                </property>
               </widget>
               <widget class="Label" name="labelImage">
                <property name="geometry">
                 <rect>
                  <x>90</x>
                  <y>0</y>
                  <width>83</width>
                  <height>84</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </widget>
              <widget class="PushButtonDone" name="pushButtonCancel">
               <property name="geometry">
                <rect>
                 <x>913</x>
                 <y>60</y>
                 <width>112</width>
                 <height>82</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <family>Nexa Light</family>
                 <pointsize>26</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
	border-image: url(:/resource/images/upload_setting/Done.png);
	border-color: rgba(255, 255, 255, 0);
}
QPushButton:hover
{
	border-image: url(:/resource/images/upload_setting/Done_OVER.png);
	border-color: rgba(255, 255, 255, 0);
}
QPushButton:pressed
{
	border-image: url(:/resource/images/upload_setting/Done.png);
	border-color: rgba(255, 255, 255, 0);
}</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="labelCancel">
               <property name="geometry">
                <rect>
                 <x>920</x>
                 <y>90</y>
                 <width>101</width>
                 <height>51</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">QLabel
{
	font: 30px &quot;Helvetica&quot;;
	color: rgb(0, 0, 0);
}
QLabel:hover
{
	font: 30px &quot;Helvetica&quot;;
	color: rgb(82, 164, 225);
}
QLabel:pressed
{
	font: 30px &quot;Helvetica&quot;;
	color: rgb(0, 0, 0);
}</string>
               </property>
               <property name="text">
                <string>Cancel</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>259</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButtonDone</class>
   <extends>QPushButton</extends>
   <header>PushButtonDone.h</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>Label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
